### DISCUSSION TITLE
Webserver but no api

### DISCUSSION THREAD

**Discussion by MojoFilter** [PlummersSoftwareLLC/NightDriverStrip#172] [Q&A]
What am I missing here that the webserver comes up but `/getEffectList` gets a 404? This is running with the `ttgo` configuration.
![image](https://user-images.githubusercontent.com/5550806/206290476-f37b94ea-4941-4372-bfff-b2a3a1d22405.png)

**Shocking Update!**
Apparently, it's more like the webserver is dying after a request? Or the webserver is just not responding frequently? I'm using a [LILYGO development board](https://www.amazon.com/LILYGO-T-Display-Arduino-Development-CH9102F/dp/B099MPFJ9M) with 16mb flash

**Comment by davepl** [comment]
That might be the current state, though last I saw it, it was able to display a current effect list and let you select from among them.  This looks older than that, though.  So it might not be up to date with my repo with the non-placeholder text.

**Reply by MojoFilter** [reply]
Interesting. I'm using the latest from main.
